### PR TITLE
[PartEditor] Prepare document file path members

### PR DESCRIPTION
**Pull request by seanshpark**
This will prepare document file path members and set filename in UI.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #641, from draft #647

UI difference before: upper, after: lower

image

**Review by jyoungyun — APPROVED** [review]
LGTM

File: src/PartEditor/PartEditor.ts [hunk 1]
```diff
@@ -80,6 +87,16 @@ export class PartEditorProvider implements vscode.CustomTextEditorProvider {
 
     this._document = document;
 
+    if (this._document) {
+      const lastSlash = this._document.fileName.lastIndexOf('/') + 1;
```
Comment (hyunsik-yoon) [inline src/PartEditor/PartEditor.ts:91]: (optional)
```suggestion
      const lastSlash = this._document.fileName.lastIndexOf(path.sep) + 1;
```

**Review by yunjayh — APPROVED** [review]
LGTM
FYI) @hyunsik-yoon is on leave rest of this week :)

**Pull request merged**